Pi2BoardPkg/AcpiTables/Common/SoftwareTpm2Table.aslc

```diff
@@ -0,0 +1,46 @@
+//  Copyright (c), Microsoft Corporation. All rights reserved.

```

Comment:
nit: Slight inconsistencies in copyright headers. Some have years, some don't.

Reply:
Removed years for consistency with most other headers.